sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -167,6 +178,14 @@ public static String encodeUrlBase64String(byte[] binaryData) {
         return encodedString;
     }
 
+    public static void configureSimpleObjectMapper(Boolean allowDuplicateProperties) {
```

Comment:
Same here, we should make it primitive to avoid null handling.

Reply:
Fixed